src/main/java/tc/oc/occ/idly/api/BaseIdlyAPI.java

```diff
@@ -0,0 +1,21 @@
+package tc.oc.occ.idly.api;
+
+import org.bukkit.entity.Player;
+import tc.oc.occ.idly.IdlyUtils;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.api.player.MatchPlayer;
+
+public class BaseIdlyAPI implements IdlyAPI {
+
+  @Override
+  public boolean isMatchRunning() {
+    return IdlyUtils.getMatch().isRunning();
+  }
+
+  @Override
+  public boolean isPlaying(Player player) {
+    Match match = IdlyUtils.getMatch();
+    MatchPlayer mp = match.getPlayer(player);
```

Comment:
This should be converted to the equivalent of just `MatchPlayer mp = PGM.get().getMatchManager().getMatchPlayer(player)`, because again, you're assuming only one match exists which doesn't necessarily have to be true